src/components/notification-manager.ts

```diff
@@ -207,8 +199,9 @@ export default class NotificationManager {
 
     checkMaxElementContraint(features: Array<Feature>) {
         if (useStore().getMaxElement() >= 0) {
-            if (features.length > useStore().getMaxElement()) {
-                this.validMaxElementConstraint = false;
+            if (features.length >= useStore().getMaxElement()) {
+                if (features.length > useStore().getMaxElement())
+                    this.validMaxElementConstraint = false;
                 this.maxElementNotificationControl?.show();
```

Comment:
I think I would show the warning only if trying to select an additional one. Else the warning is not so visible once the select does not work
To check with Jonathan

Reply:
done